app/code/Magento/Contact/Api/ConfigInterface.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Contact module base controller
+ */
+namespace Magento\Contact\Api;
+
+/**
+ * Contact module configuration
+ */
+interface ConfigInterface
```

Comment:
This interface is neither Service Interface (does not describe operations on data) nor Data Interface (it does not consist only of getters and setters for the fields). Also, it does not looks like it makes sense to expose it as Web API.

Let's move these two interfaces out of Api namespace to the Model. Please add @api annotation to the interface to indicate that this is backward compatible module's interface.